### ISSUE TITLE
Panel with leafmap not working 

### ISSUE THREAD

**Issue by sophiamyang** [holoviz/panel#3134]
Hi, I'm trying to make a Panel app with leafmap. 

Here is my [Code](https://github.com/sophiamyang/panel_leafmap/blob/main/panel_leafmap.ipynb) and [environment](https://github.com/sophiamyang/panel_leafmap/blob/main/environment.yml).

It seems to work correctly in my notebook:
![example](https://user-images.githubusercontent.com/39501859/150658392-bac3e698-ec1f-4012-84ab-b07be680db98.gif)

However, when I tried to do panel serve or template.show(), I got the following error message:

![import leafmap](https://user-images.githubusercontent.com/39501859/150658382-e3a2968b-721b-41fd-82b8-9e79f9ea0221.png)

Then I tried the folium backend`import leafmap.foliumap as leafmap`,  I got this error message: 
![import leafmap.foliumap as leafmap](https://user-images.githubusercontent.com/39501859/150658383-9b46d41b-31bb-4dcb-b555-10d06337aea7.png)

Can I please get some help on this? Thanks!

**Comment by jbednar** [comment]
Try `conda install -c bokeh ipywidgets_bokeh`?

**Comment by sophiamyang** [comment]
yay that works thanks! I didn't know ipywidgets_bokeh was in the bokeh channel. thanks!

**Comment by jbednar** [comment]
Yeah, we do need to get that onto more common channels like conda-forge and defaults.

**Comment by rh-afk** [comment]
I just try @sophiamyang [code](https://github.com/sophiamyang/panel_leafmap/blob/main/panel_leafmap.ipynb) unfortunately I got error when serve the panel however if I run interactively inside notebook there was run smoothly only when we serve as panel its error. I use panel 1.2.0, bokeh 3.2.1,ipywidgets-bokeh 1.5.0
Error message:
Error running application handler <bokeh.application.handlers.notebook.NotebookHandler object at 0x7f3f9930d580>: module 'comm' has no attribute 'DummyComm'
File 'ipywidget.py', line 70, in _on_widget_constructed:

did any one get the same issue?

**Comment by philippjfr** [comment]
Could you report your version of the `comm` package?

**Comment by rh-afk** [comment]
Hi @philippjfr  I use 
comm                        0.1.2
pyviz-comms                 2.0.2

Thanks

**Comment by rh-afk** [comment]
Hi @philippjfr  after I update to the latest comm(0.1.4) and pyviz(2.3.2), now panel show up but only the header while the map still empty
![Capture](https://github.com/holoviz/panel/assets/141318843/5cf36b73-782b-4d17-90b8-c3d567d2a15d)

**Comment by rh-afk** [comment]
Hi @philippjfr  did you get the same result(can reproduce my problem) or only me got that result.
thanks
